packages/@atjson/offset-annotations/src/annotations/cne-ticketing-widget-embed.ts

```diff
@@ -0,0 +1,8 @@
+import { BlockAnnotation } from "@atjson/document";
+
+export class CneTicketingWidgetEmbed extends BlockAnnotation<{
+  url: string;
+}> {
+  static vendorPrefix = "offset";
+  static type = "cne-ticketing-widget-embed";
```

Comment:
We've been going back and forth on naming. For sure I think we don't need `widget` in the annotation name. Perhaps
`cne-ticketing-embed` or `cne-swoogo-embed` ?

Reply:
@bachbui We prefer cne-event-registraton-embed

code to insert into CKeditor in copilot
<cne-event-registration url="https://baseurl?loggedout=loggedouturlslug&loggedin=loggedinslug&privacy=true"></cne-event-registration>

markdown
[#cneeventregistration: https://baseurl?loggedout=loggedouturlslug&loggedin=loggedinslug&privacy=true]

embed
<cne-event-registration-embed url="https://baseurl?loggedout=loggedouturlslug&loggedin=loggedinslug&privacy=true"></cne-event-registration-embed>

Is that okay? I wait for a response before proceeding with the changes (file names, classes, tests, etc.)

Reply:
Thanks sounds great to me

Reply:
@bachbui rename cne-ticketing-widget to cne-event-registration